src/Handler/QueryMonitorHandler.php

```diff
@@ -0,0 +1,55 @@
+<?php # -*- coding: utf-8 -*-
+/*
+ * This file is part of the Wonolog package.
+ *
+ * (c) Inpsyde GmbH
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Inpsyde\Wonolog\Handler;
+
+use Monolog\Handler\AbstractProcessingHandler;
+use Psr\Log\LogLevel;
+
+/**
+ * Class QueryMonitorHandler.
+ *
+ * @package BoxUk\Plugins\Base
```

Comment:
C&P mistake?

Reply:
ah yep, opened this after opening https://github.com/boxuk/wp-project-skeleton/pull/169 but will get this removed 👍

Reply:
Sorted.